terraform/ratingservice/01_project.tf

```diff
@@ -24,4 +24,10 @@ resource "google_project_service" "cloudscheduler" {
   service                    = "cloudscheduler.googleapis.com"
   disable_dependent_services = true
   project                    = var.gcp_project_id
-}
\ No newline at end of file
+}
+
+resource "google_project_service" "cloudbuild" {
+  service                    = "cloudbuild.googleapis.com"
+  disable_dependent_services = true
+  project                    = var.gcp_project_id
+}
```

Comment:
nit: why to add activation of Cloud Build?

Reply:
It looks like app engine uses cloud build to deploy the instance. 

I was sometimes seeing issues where App Enggine deployment would fail unexpectedly, and I think it was because some of the necessary APIs weren't ready yet

Reply:
Just checked and saw that the Cloud Build API in new projects is not activated. Indeed, it makes sense to activate it before AppEngine will do it. It improves execution time and might resolve the corner cases that you mentioned.